ocean_provider/serializers.py

```diff
@@ -43,23 +48,14 @@ def serialize(self):
 
         dict_template["id"] = algorithm_did
         dict_template["rawcode"] = ""
-
-        asset_urls = get_service_files_list(sa, self.provider_wallet)
-        asset_url = asset_urls[0] if asset_urls else None
-        if asset_url:
-            asset_url = append_userdata(asset_url, self.algo_data, "algouserdata")
-            dict_template["url"] = asset_url
-        else:
-            dict_template["remote"] = {
-                "serviceEndpoint": self.algo_service.service_endpoint,
-                "txId": algorithm_tx_id,
-                "serviceId": self.algo_service.id,
-            }
-
-            userdata = self.algo_data.get("algouserdata")
-            if userdata:
-                dict_template["remote"]["algouserdata"] = userdata
-
         dict_template["container"] = algo_asset.metadata["algorithm"]["container"]
+        dict_template["remote"] = {
+            "serviceEndpoint": self.algo_service.service_endpoint,
+            "txId": algorithm_tx_id,
+            "serviceId": self.algo_service.id,
+        }
+        # userdata = self.algo_data.get("algouserdata")
```

Comment:
why is this commented out? Should we remove it entirely? Otherwise please leave a comment indicating why it's here and commented.

Reply:
fixed